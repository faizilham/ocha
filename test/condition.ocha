let a = 1;

if (a < 10) {
    print("a");                 // a
    print("a1");                // a1

    if (a > 1) {
        print("c");
        print("c1");
    } else if (a < 5) {
        print("d");             // d
        print("d1");            // d1
    } else {
        print("e");
    }
} else {
    print("b");
    print("b1");
}

print("end");                   // end
