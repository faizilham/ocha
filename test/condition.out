a
a1
d
d1
end
